required/tools/verbatim.dtx

```diff
@@ -795,7 +795,7 @@
   \obeylines
 %    \end{macrocode}
 %    change the category code of all special characters,
-%    to $12$ (other).
+%    to $12$ (other),
```

Comment:
Better to use ":" here.

Reply:
Note that this is continued with “and switch to the font to be used”, so we have a sentence interspersed with code.

Reply:
Not a nice thing to do!  Is there a comma after the code too:-)? 

Or is it possible to start a new sentence after the code?